### PR TITLE
NIFI-4855: The layout of NiFi API document is broken

### PR DESCRIPTION
**Pull request by tasanuma**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by tasanuma** [comment]
This pull-request uses _**notes**_ variables instead of _**value**_ variables for long sentences.

Before: https://issues.apache.org/jira/secure/attachment/12909857/broken_ui.png
After: https://issues.apache.org/jira/secure/attachment/12912700/fixed_ui.001.png

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/resources/templates/index.html.hbs [hunk 1]
```diff
@@ -675,6 +676,14 @@
             </div>
             <div id="system-diagnostic-endpoints" class="section-endpoints hidden"></div>
         </div>
+        <div class="section">
+            <div class="section-header">
+                <div class="title link">Versions</div>
+                <div class="sub-title section-description">Versioning</div>
```
Comment (zenfenan) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/resources/templates/index.html.hbs]: Let's try a different one that's a little descriptive here, like "Control flow versioning" or "Save, delete manage flow versioning" or something better?

Comment (tasanuma) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/resources/templates/index.html.hbs]: Thanks for the review, @zenfenan! Updated the pull-request addressing your comments.

**Comment by mcgilman** [comment]
Will review...

**Comment by mcgilman** [comment]
Thanks @tasanuma! This has been merged to master.

**Comment by tasanuma** [comment]
Thanks for reviewing and committing it, @mcgilman!

**Pull request closed without merging**